src/models.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 const debug = require("debug")("oasis");
-const { isRoot, isReply } = require("ssb-thread-schema");
+const { isRoot, isReply: isComment } = require("ssb-thread-schema");
```

Comment:
I find Object destructuring assignments to be counterintuitive so I had to look this up (I always think they work the opposite way around).

In this case we're taking `isReply` from `ssb-thread-schema` and assigning it to a local variable `isComment`.

Maybe a comment could help others avoid my confusion
`// Rename ssb-thread-schema's isReply to isComment